sdk/translation/Azure.AI.Translation.Document/tests/samples/SampleSnippets.cs

```diff
@@ -5,15 +5,21 @@
 using System.Collections.Generic;
 using Azure.AI.Translation.Document.Tests;
 using Azure.Core.TestFramework;
+using Azure.Identity;
 using NUnit.Framework;
 
 namespace Azure.AI.Translation.Document.Samples
 {
     /// <summary>
     /// Samples that are used in the associated README.md file.
     /// </summary>
-    public partial class SampleSnippets : SamplesBase<DocumentTranslationTestEnvironment>
+    public partial class SampleSnippets : DocumentTranslationLiveTestBase
```

Comment:
looking at the sample code we have here, we don't actually need for this class to inherit from `DocumentTranslationLiveTestBase`. We can just leave it as it